pkg/persistence/account/writer/writer.go

```diff
@@ -0,0 +1,169 @@
+/*
+ * @license
+ * Copyright 2023 Dynatrace LLC
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package writer
+
+import (
+	"fmt"
+	"github.com/dynatrace/dynatrace-configuration-as-code/v2/internal/log"
+	"github.com/dynatrace/dynatrace-configuration-as-code/v2/pkg/account"
+	persistence "github.com/dynatrace/dynatrace-configuration-as-code/v2/pkg/persistence/account"
+	"github.com/spf13/afero"
+	"gopkg.in/yaml.v2"
+	"path/filepath"
+)
+
+type Context struct {
+	Fs            afero.Fs
+	OutputFolder  string
+	ProjectFolder string
+}
+
+func WriteAccountResources(writerContext Context, resources account.Resources) error {
```

Comment:
```suggestion
func Write(writerContext Context, resources account.Resources) error {
```

I'd go with the simple wording. The import will probably called `accountWriter`, then the call would be `accountWriter.Write()`. IIRC the other loads have the same structure as well.